telegram/ext/updater.py

```diff
@@ -251,11 +255,54 @@ def start_polling(self,
         Returns:
             :obj:`Queue`: The update queue that can be filled from the main thread.
 
+        Raises:
+            ValueError: If :attr:`clean` is set as :obj:`datetime.timedelta` and is < 1 second.
+
         """
         with self.__lock:
             if not self.running:
                 self.running = True
 
+                self.logger.debug('clean:               "%s"', type(clean))
+                self.logger.debug('clean:               "%s"', clean)
+                self.logger.debug('now:                 "%s"', datetime.now(tz=timezone.utc))
+
+                if isinstance(clean, timedelta):
+                    if clean.total_seconds() < 1:
```

Comment:
Maybe consider `abs(clean.total_seconds())` here and subtract the absolute value below. People might not be sure whether they need to pass a positive or a negative timedelta. Also add a note to the doc string stating that the sign doesn't matter.

Reply:
I will do because you request it but under formal protest :) imhoo the user should use timedelta correctly and be warned if they don't.

Reply:
resolved with 33410d3 and bd21699